Kernel/Arch/aarch64/init.cpp

```diff
@@ -93,8 +93,23 @@ void __stack_chk_fail()
     Prekernel::halt();
 }
 
-[[noreturn]] void __assertion_failed(char const*, char const*, unsigned int, char const*)
+[[noreturn]] void __assertion_failed(char const* msg, char const* file, unsigned line, char const* func)
 {
+    auto& uart = Prekernel::UART::the();
+
+    uart.print_str("\r\n\r\nAssertion failed in kernel");
+    
+    uart.print_str("\r\nMsg: ");
+    uart.print_str(msg);
+
+    uart.print_str("\r\nFile: ");
+    uart.print_str(file);
+    uart.print_str(":");
+    uart.print_num(line);
+    
+    uart.print_str(" ");
+    uart.print_str(func);
```

Comment:
Let's use the same format as the x86 kernel here, there's no reason not to :)

https://github.com/SerenityOS/serenity/blob/086969277e74d8ba065bf8145d3aeb0dec0bfee5/Kernel/Arch/x86/common/CPU.cpp#L17-L18

Reply:
Done